tests/core/test_policies.py

```diff
@@ -407,35 +408,43 @@ async def test_gen_batch(self, trained_policy: TEDPolicy, default_domain: Domain
             training_data, label_ids, entity_tags, all_labels
         )
         batch_size = 2
-        print(len(next(model_data._gen_batch(batch_size=batch_size))))
-        print()
```

Comment:
cc @Ghostvv So that the `e2e` PR is not merged with these print statements.

Reply:
already fixed in `e2e`